src/Docker/Runner/Limits.php

```diff
@@ -78,17 +78,17 @@ public function getCpuLimit(Image $image)
         if (in_array(self::DYNAMIC_BACKEND_JOBS_FEATURE, $this->projectFeatures)) {
             switch ($this->containerType) {
                 case null:
-                case 'small':
+                case 'xsmall':
                     $cpuLimit = 1;
                     break;
-                case 'medium':
+                case 'small':
                     $cpuLimit = 2;
                     break;
-                case 'large':
+                case 'medium':
                     $cpuLimit = 4;
                     break;
-                case 'xlarge':
-                    $cpuLimit = 16;
+                case 'large':
+                    $cpuLimit = 14;
```

Comment:
to ze zmizel `xlarge` a naopak pribyl `xsmall` je cajk?

Reply:
Jojo v původním návrhu to bylo small až xlarge, nově je to xsmall až large -  https://keboola.atlassian.net/browse/KBCP-381 https://keboola.atlassian.net/wiki/spaces/KB/history/2619768896/Python+R+Data+Science+Jobs+Backend+Billing